components/SockJsOrderClient.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'react'
+import {apiRoot} from "../constants/Backend";
+import SockJsClient from 'react-stomp';
+import * as Device from 'expo-device';
+import {playSound} from "../helpers/playSoundHelper";
+
+
+const url = `${apiRoot}/ws`
+
+class SockJsClientOrderDisplayBase extends Component {
+  constructor(props, context) {
+    super(props, context)
+  }
+
+  render() {
+    const {
+      meta,
+      topics,
+      handleSockClientData,
+      handleConnect,
+      handleDisConnect,
+      clientId,
+      debugMode,
+    } = this.props;
+
+
+    return (
+      <SockJsClient url={url} topics={[`${topics}${clientId}`]}
+        onMessage={(data) => {
+          if (data?.needAlert) {
+            playSound()
+          }
```

Comment:
I think this logic is specific to order display screen and therefore should be moved to order display screen logic.